main.js

```diff
@@ -31,22 +30,45 @@ hlDefault.use(new defaultCodecs.MultibaseBase58btc());
  * @param {Object} options - The options for the create operation.
  * @param {Uint8Array} options.data - The data associated with the given URL. If
  *   provided, this data is used to create the cryptographic hash.
- * @param {Array} options.urls - One or more URLs that contain the data
+ * @param {Array} [options.url] - A single URL that contain the data
+ *   referred to by the hashlink.
+ * @param {Array} [options.urls] - Multiple URLs that contain the data
  *   referred to by the hashlink.
- * @param {Array} options.codecs - One or more URLs that contain the data
+ * @param {Array} [options.codecs] - One or more URLs that contain the data
  *   referred to by the hashlink.
- * @param {Object} options.meta - A set of key-value metadata that will be
+ * @param {Object} [options.meta={}] - A set of key-value metadata that will be
  *   encoded into the hashlink.
  *
  * @returns {Promise<string>} Resolves to a string that is a hashlink.
  */
-async function create({data, urls, url,
-  codecs = ['mh-sha2-256', 'mb-base58-btc'], meta = {}}) {
+async function create(options) {
```

Comment:
-1 to this change... requires the use of too much boilerplate. Rename `urls` to `url` instead and allow it to be either a string or an array of URLs.

Reply:
Understood. I undid the change, and renamed `urls` to `url`.